posts/TEMPLATE/index.md

```diff
@@ -1,9 +1,115 @@
 ---
 date: 2030-12-31 # Auto updates on publish.
-title: Hello World!
-author: sascha.lampalzer@cybertec.at
-tags: ["list-tags-here", "🙂"] # max. 10 tags; lowercase; dash-separated
-description: "This is an example" # max. 300 chars.
+title: Quickly test whether ports can be reached on your linux machine.
```

Comment:
```suggestion
title: Quickly test whether ports can be reached on your linux machine
```

Reply:
Last remark before approving: Headings shouldn't really be sentences, therefore removing the dot 🙂

Reply:
ok, fine :)